packages/services/src/competition.service.ts

```diff
@@ -648,7 +609,7 @@ export class CompetitionService {
   ): Promise<string[]> {
     const competitionAgents = await this.agentService.getAgentsForCompetition(
       competitionId,
-      { sort: "", limit: 1000, offset: 0 },
+      { sort: "", limit: 1000, offset: 0, includeInactive: false },
```

Comment:
created a follow up for the hardcoded "limit" because iirc there are a few places we do think. not a big deal rn.

https://github.com/recallnet/js-recall/blob/fbd79d7ca3dd9da12aaf1e74bec18df236a7adef/packages/services/src/competition.service.ts#L649